LayoutTests/platform/glib/damage/image-as-layer-src-change.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="./common.css">
+    <style>
+      canvas {
+          display: block;
+      }
+      img {
+          position: fixed;
+          left: 0px;
+          top: 50px;
+      }
+    </style>
+    <script src="./common.js"></script>
+    <script>
+      async function takeCanvasSnapshotAsBlobURL() {
+          return new Promise(async (resolve) => {
+              var canvas = document.getElementsByTagName("canvas")[0];
+              canvas.toBlob((blob) => {
+                  resolve(URL.createObjectURL(blob));
+              });
+          });
+      }
```

Comment:
There are two tests, for now, that use `takeCanvasSnapshotAsBlobURL()`. It's almost justified if we were to move it into `common.js`, I think.

In the method, the canvas is constant,  I think, and doesn't feel like it needs to be inside the promise. The promise itself can be made less verbose -- but thatś probably a matter of personal preference. I don't think you need to specify `async` inside again

```suggestion
      async function takeCanvasSnapshotAsBlobURL() {
          const canvas = document.getElementsByTagName("canvas")[0];
          return new Promise(resolve =>
              canvas.toBlob(blob => resolve(URL.createObjectURL(blob)))
          );
      }
```